libs/cardano-ledger-test/src/Test/Cardano/Ledger/Examples/TwoPhaseValidation.hs

```diff
@@ -2171,11 +2177,7 @@ alonzoUTXOWexamplesB pf =
               pf
               (trustMeP pf True $ missing1phaseScriptWitnessTx pf)
               ( Left
-                  [ fromUtxos @era . CollectErrors $
-                      [ NoRedeemer (Spending (mkGenesisTxIn 100)),
-                        NoWitness (timelockHash 0 pf)
-                      ],
```

Comment:
I'm really happy that the cleanup that I did to `collectTwoPhaseScriptInputs` also cleaned up these cascading predicate failures. This was "parse, don't validate" at its best!